app/Http/Controllers/HomeController.php

```diff
@@ -23,14 +24,20 @@ public function __construct()
      *
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(Request $request)
     {
+        $count = isset($request['items_per_page']) ? $request['items_per_page'] : 5;
```

Comment:
Please use laravel request `$count = $request->get('items_per_page', 5);` to avoid unnecessary ternary conditions.